tests/syntax/test_parser.py

```diff
@@ -184,13 +184,13 @@ def test_specifier_beyond(self):
                 assert False
 
     def test_specifier_beyond_from(self):
-        mod = parse_string_helper("new Object beyond position by distance from base")
```

Comment:
The old test now means `new Object beyond position by (distance from base)` where `distance from` is a prefix operator

Reply:
Hmm, I would like `distance` to be a valid variable name, if possible. Can we set up the precedence so that this parses as `new Object beyond (position) by (distance) from (base)`, or would that break other natural cases?

Reply:
Per our discussion, we'll leave the behavior as it is for now; users will need to introduce parentheses if they want to split `distance` from `from base`.